global.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sdk":  {
+                "version":  "2.2.108"

```

Comment:
Nit: The test project is .NET Core 2.1, not 2.2 (though 2.2 SDK should work fine).

Reply:
It's just the SDK I care about, since the Carbon assemblies target netstandard2.0.